app/scripts/models/account.js

```diff
@@ -414,10 +416,13 @@ define(function (require, exports, module) {
         }
       })
       .then((updatedSessionData) => {
+        this._metrics.logEvent('core.signin.success');
```

Comment:
> I don't even know which one is anymore

Yea I agree it is a beast to navigate. I don't know if we should necessarily take out others because those are the fine grain metrics on each screen. These new metrics sum of the events from all screens.

That being said I wonder if it even needed now that we have signin/signup flowEvents? These metrics really only benefit Datadog graphs, cc @rfk